mingus/core/chords.py

```diff
@@ -109,61 +110,82 @@
-    "11": " eleventh",
-    "add11": " eleventh",
-    "sus4b9": " suspended fourth ninth",
-    "susb9": " suspended fourth ninth",
-    "m7": " minor seventh",
-    "M7": " major seventh",
-    "dom7": " dominant seventh",
-    "7": " dominant seventh",
-    "m7b5": " half diminished seventh",
-    "dim7": " diminished seventh",
-    "m/M7": " minor/major seventh",
-    "mM7": " minor/major seventh",
-    "m6": " minor sixth",
-    "M6": " major sixth",
-    "6": " major sixth",
-    "6/7": " dominant sixth",
-    "67": " dominant sixth",
-    "6/9": " sixth ninth",
-    "69": " sixth ninth",
-    "9": " dominant ninth",
-    "add9": " dominant ninth",
-    "7b9": " dominant flat ninth",
-    "7#9": " dominant sharp ninth",
-    "M9": " major ninth",
-    "m9": " minor ninth",
-    "7#11": " lydian dominant seventh",
-    "m11": " minor eleventh",
-    "M13": " major thirteenth",
-    "m13": " minor thirteenth",
-    "13": " dominant thirteenth",
-    "add13": " dominant thirteenth",
-    "7b5": " dominant flat five",
-    "hendrix": " hendrix chord",
-    "7b12": " hendrix chord",
-    "5": " perfect fifth",
-}
+                             # Sixths Ninths Elevenths Thirteenths Altered
+                             # Chords Special
+    'm': ' minor triad',
+    'M': ' major triad',
+    '': ' major triad',
+    'dim': ' diminished triad',
+    'aug': ' augmented triad',
+    '+': ' augmented triad',
+    '7#5': ' augmented minor seventh',
+    'M7+5': ' augmented minor seventh',
+    'M7+': ' augmented major seventh',
+    'm7+': ' augmented minor seventh',
+    '7+': ' augmented major seventh',
+    'sus47': ' suspended seventh',
+    '7sus4': ' suspended seventh',
+    'sus4': ' suspended fourth triad',
+    'sus2': ' suspended second triad',
+    'sus': ' suspended fourth triad',
+    '11': ' eleventh',
+    'add11': ' eleventh',
+    'sus4b9': ' suspended fourth ninth',
+    'susb9': ' suspended fourth ninth',
+    'm7': ' minor seventh',
+    'M7': ' major seventh',
+    'dom7': ' dominant seventh',
+    '7': ' dominant seventh',
+    'm7b5': ' half diminished seventh',
+    'm11b5': ' minor eleven flat five',
+    'dim7': ' diminished seventh',
+    'm/M7': ' minor/major seventh',
+    'mM7': ' minor/major seventh',
+    'm6': ' minor sixth',
+    'M6': ' major sixth',
+    '6': ' major sixth',
+    '6/7': ' dominant sixth',
+    '67': ' dominant sixth',
+    '6/9': ' sixth ninth',
+    '69': ' sixth ninth',
+    '9': ' dominant ninth',
+    'add9': ' dominant ninth',
+    '7b9': ' dominant flat ninth',
+    '7#9': ' dominant sharp ninth',
+    'M9': ' major ninth',
+    'm9': ' minor ninth',
+    '7#11': ' lydian dominant seventh',
+    'm11': ' minor eleventh',
+    'M13': ' major thirteenth',
+    'm13': ' minor thirteenth',
+    '13': ' dominant thirteenth',
+    'add13': ' dominant thirteenth',
+    '7b5': ' dominant flat five',
+    'hendrix': ' hendrix chord',
+    '7b12': ' hendrix chord',
+    '5': ' perfect fifth',
```

Comment:
You should run `make format` prior to your commits to avoid accidentally committing undesired coding style changes.